src/prostredi/editor/editor.md

```diff
@@ -0,0 +1,19 @@
+# Vývojové prostředí
+
+Abychom mohli přeložit a spustit nějaký program, musíme ho obvykle nejprve zapsat do
+jednoho nebo více souborů ve formě tzv. **zdrojového kódu** (*source code*). K usnadnění tohoto procesu
+existují **textové editory** a **vývojová prostředí** jako například `MS Visual Studio`, `QtCreator`, `JetBrains CLion`,
+`CodeBlocks`, `Visual Studio Code`, `vim`, `emacs` apod. Tyto programy usnadňují psaní kódu pomocí zvýrazňování
+syntaxe, automatizace překladu, spouštění a testování programů a také správy projektů.
+
+Na cvičeních UPR budeme používat editor `Visual Studio Code`, který je
+[dostupný zdarma](https://code.visualstudio.com/). [Zde](vscode.md) je stručný návod k jeho použití. Při
+programování se hodí detailně znát a efektivně využívat editor, který používáte, ale pro začátek
+nám budou stačit naprosté základy.
+
+Jako alternativu lze použít [CLion](clion/clion.md), plnohodnotné **IDE** (*Integrated Development Environment*) s pokročilými funkcemi, které usnadňují vývoj v C a C++. Nabízí nativní integraci s build systémem [CMake](clion/cmake.md), což
+vývojářům umožňuje snadnou správu projektů a konfigurací bez nutnosti složitého nastavování.
+
+Jednou z hlavních výhod CLionu je jeho jednotné uživatelské rozhraní a workflow, které sdílí s dalšími produkty od JetBrains. Jakmile se naučíte pracovat s CLionem, přechod na jiné nástroje jako
```

Comment:
Tohle už vypadá vyloženě jako reklama na IntelliJ :laughing:

Reply:
Fixed in https://github.com/mrlvsb/upr-skripta/pull/94/commits/666167a62a657241f65fa318d046a08e8aeb4710